deploy/overlays/dev-template/config.yaml

```diff
@@ -4,11 +4,6 @@ kind: JBSConfig
 metadata:
   name: jvm-build-config
 spec:
-  owner: QUAY_USERNAME
-  relocationPatterns:
-    - relocationPattern:
-        buildPolicy: "default"
-        patterns:
-          - pattern:
-              from: (io.github.stuartwdouglas.hacbs-test.simple):(simple-jdk17):(99-does-not-exist)
-              to: io.github.stuartwdouglas.hacbs-test.simple:simple-jdk17:0.1.2
+  registry:
+    owner: QUAY_USERNAME
```

Comment:
This is the same change as in https://github.com/redhat-appstudio/jvm-build-service/pull/880/files#diff-d4012663d2198be9ac5255ec8dcd2078a1a65dec31b3ffb4ce11f36fea664b71 so will conflict. 
Is relocationPatterns no longer required?

Reply:
I am going to remove this before it goes in. I was having some issues with my cluster and I was wondering if this was the cause.

Reply:
relocationPatterns was never really required, it was there for testing, and I don't know if we are ever going to really use it.